sdk/python/kfp/__main__.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ._client import Client
+import sys
+import subprocess
+import pprint
+import time
+import json
+import click
+
+from tabulate import tabulate
+
+@click.group()
```

Comment:
Maybe add a todo to expose other client APIs such as list_experiment, etc.?

Reply:
sg, will add